.github/workflows/dotnet.yml

```
@@ -0,0 +1,32 @@
+# This workflow will build a .NET project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
+
+name: .NET
+permissions:
+  contents: read
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: 10.0.x
+    - name: Restore dependencies
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --no-restore
+    - name: Test
+      run: dotnet test --no-build --verbosity normal
+      env:
+        JWT__SecretKey: ${{ secrets.JWT_SECRETKEY }}
```

```suggestion
        JWT__SecretKey: ${{ secrets.JWT_SECRETKEY }}
        GoogleMaps__ApiKey: ${{ secrets.GOOGLE_MAPS_API_KEY }}
        Google__ClientId: ${{ secrets.GOOGLE_CLIENT_ID }}
```
        Forward all env vars for tests